pkg/apis/pipeline/v1/task_validation_test.go

```diff
@@ -1925,12 +1925,13 @@ func TestTaskSpecBetaFields(t *testing.T) {
 	}}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if err := tt.spec.Validate(context.Background()); err == nil {
+			task := v1.Task{ObjectMeta: metav1.ObjectMeta{Name: "foo"}, Spec: tt.spec}
+			if err := task.Validate(context.Background()); err == nil {
```

Comment:
Should we rename the test name from "TestTaskSpecBetaFields" to "TestTaskBetaFields"?

Reply:
Done